tests/fuzzer/flatbuffers_scalar_fuzzer.cc

```diff
@@ -0,0 +1,380 @@
+#include <stddef.h>
+#include <stdint.h>
+#include <algorithm>
+#include <clocale>
+#include <memory>
+#include <regex>
+#include <string>
+
+#include "flatbuffers/idl.h"
+
+#define flags_scalar_type 0x0F  // type of scalar value
+#define flags_quotes_kind 0x10  // quote " or '
+#define flags_json_bracer 0x20  // {} or []
+#define flags_0x40 0x40         // reserved flag
+#define flags_clocale 0x80      // change default C-locale
+
+#define TEST_OUTPUT_LINE(...)     \
+  {                               \
+    fprintf(stderr, __VA_ARGS__); \
+    fprintf(stderr, "\n");        \
+  }
+
+static void TestFail(const char *expval, const char *val, const char *exp,
```

Comment:
this PR that is about to land has factored out these test macros, maybe you could use the same? https://github.com/google/flatbuffers/pull/4902